rdbms/src/main/java/org/spine3/server/storage/jdbc/query/factory/CommandStorageQueryFactory.java

```diff
@@ -0,0 +1,98 @@
+package org.spine3.server.storage.jdbc.query.factory;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.spine3.base.CommandId;
+import org.spine3.base.CommandStatus;
+import org.spine3.base.Error;
+import org.spine3.base.Failure;
+import org.spine3.server.storage.CommandStorageRecord;
+import org.spine3.server.storage.jdbc.query.constants.CommandTable;
+import org.spine3.server.storage.jdbc.query.tables.commands.*;
+import org.spine3.server.storage.jdbc.util.DataSourceWrapper;
+import org.spine3.server.storage.jdbc.util.IdColumn;
+
+public class CommandStorageQueryFactory {
+
+    private final IdColumn idColumn;
+    private final DataSourceWrapper dataSource;
+
+    public CommandStorageQueryFactory(DataSourceWrapper dataSource) {
+        this.idColumn = new IdColumn.StringIdColumn();
+        this.dataSource = dataSource;
+    }
+
+    private static Logger log() {
+        return LogSingleton.INSTANCE.value;
+    }
+
+    private enum LogSingleton {
+        INSTANCE;
+        @SuppressWarnings("NonSerializableFieldInSerializableClass")
+        private final Logger value = LoggerFactory.getLogger(CommandStorageQueryFactory.class);
+    }
+
+    public CreateTableIfDoesNotExistQuery getCreateTableIfDoesNotExistQuery(){
```

Comment:
Rename to `newBlaBla...`. Such methods starting with `get` can be confused with getters, when they are factory methods.